src/devices/ovrheadset/OVRHeadset.h

```diff
@@ -64,6 +82,11 @@ class OVRHeadset : public yarp::dev::DeviceDriver,
     static void glfwErrorCallback(int error, const char* description);
     static void ovrDebugCallback(uintptr_t userData, int level, const char* message);
     static void DebugHmd(ovrHmdDesc hmdDesc);
+    inline void errorManager(ovrResult error);
+    inline void fillAxisStorage();
+    inline void fillErrorStorage();
+    inline void fillButtonStorage();
+    inline void fillHatStorage();
```

Comment:
This should be fine in this specific case since the methods are private, but it is dangerous in the general case... when you declare a function `inline` you should have the implementation in the same file, or you risk to get an "unresolved external" error from the linker.

To be honest, I don't think that the compiler here will `inline` these functions, since their definition is not that short, so I'd just remove the inline.